### DISCUSSION TITLE
Define Filters with AND and OR concurrent

### DISCUSSION THREAD

**Discussion by Kakovisk** [Budibase/budibase#9664] [Help] [answered]
Hi everybody!

I need to filter in Data Provider conditions with AND and OR but when I Defined Filter I just have two options: Match All Filter and Match Any Filter. Someone can be help me?

I would liked of some similar in sql: "where (x=y or x=a) and c=t "

**Comment by melohagan** [comment, marked as answer]
Hey @Kakovisk 

Currently you cannot filter with a combination of AND + OR. Issue here: https://github.com/Budibase/budibase/issues/3183

If you are using SQL or another external database, then you could add any conditions to your query: https://docs.budibase.com/docs/custom-queries

**Reply by Kakovisk** [reply]
Tks @melohagan
